setup.py

```diff
@@ -126,7 +126,7 @@ def run(self):
     packages=find_packages(
         exclude=["docs", "docs-src", "tests", "tests.*", "tutorial"]
     ),
-    install_requires=["aiohttp>3.5.2"],
+    install_requires=["aiohttp>3.5.2,<4.0.0"],
```

Comment:
4.0.0 alpha releases don't work with our current test code. we can check this issue again when they release beta/release candidate versions.